### ISSUE TITLE
oscap-docker mount container in current directory

### ISSUE THREAD

**Issue by ybznek** [OpenSCAP/openscap#329]
I think, we should mount container in /tmp or in some directory which is intended to mounting.

```
zmoravec: /proc 13:37:41
$ sudo oscap-docker image-cve rhel7
Traceback (most recent call last):
  File "/bin/oscap-docker", line 108, in <module>
    args.func()
  File "/bin/oscap-docker", line 44, in cve_scan
    result = OS.scan_cve(self.args.scan_target, self.unknown_args)
  File "/usr/lib/python2.7/site-packages/oscap_docker_python/oscap_docker_util.py", line 193, in scan_cve
    _tmp_mnt_dir = DM.mount(image)
  File "/usr/lib/python2.7/site-packages/Atomic/mount.py", line 322, in mount
    driver_mount_fn(identifier, options)
  File "/usr/lib/python2.7/site-packages/Atomic/mount.py", line 368, in _mount_devicemapper
    raise MountError(e)
Atomic.mount.MountError: [Errno 2] No such file or directory: '/proc/024f294ea48f3e3b79f9'
```
I cannot run oscap-docker in ```/proc``` directory, because oscap-docker want to mount the image to ```/proc/024f294ea48f3e3b79f9```.

**Comment by mpreisler** [comment]
Is this an issue only in `oscap-docker` CLI tool or does the same thing apply to other tools using the `oscap_docker_python` package?

**Comment by ybznek** [comment]
It seems, that problem is probably here https://github.com/OpenSCAP/openscap/blob/maint-1.2/utils/oscap_docker_python/oscap_docker_util.py#L180 ```mnt_dir=os.getcwd()```

**Daemon:**
Daemon doesn't use ```OscapScan```
openscap_daemon/cve_scanner/scan.py:        self.DM = DockerMount("/tmp", mnt_mkdir=True)
TLDR: daemon doesn't have this problem.

In my opinion, easiest solution is replace ```os.getcwd()``` with ```/tmp```, so oscap_docker will use same directory for mount as daemon.

**Comment by mpreisler** [comment]
Alright. Are we sure this is safe? Can I guess the directory that will be created in `/tmp` and break it by creating it?

What I am getting at is that maybe we should be using private temp dir. It's very little added effort and we are guaranteed a directory that is private to whoever created it.

**Comment by ybznek** [comment]
Sorry, you are right. It is better to have private directory for mount in oscap-docker and daemon.

**Comment by mpreisler** [comment]
OK. @ybznek, are you planning to fix this yourself? I volunteer to review the PRs to both projects.

**Comment by ybznek** [comment]
I will, but need to finish some other issues, first.

**Comment by ybznek** [comment]
I've created PRs to oscap-docker. https://github.com/OpenSCAP/openscap/pull/363
Personally I don't know why mnt_dir is parameter of constructor, because container is not accessible after function ends.

**Comment by jan-cerny** [comment]
Fixed in #363

**Comment by ybznek** [comment]
Sorry, I have delay (out of office) between solving these parts of issues. I would like to do fixes in daemon too.

**Comment by ybznek** [comment]
fixed
